docs/guides/reference/collection_3_summary_grid.md

```diff
@@ -25,27 +25,31 @@ in an effort to reduce:
 * size on disk,
 * lossy re-projection of data from native coordinate reference system.
 
-To make information across Australia comparable, continental-scale, 
+To make information across continental-Australia comparable,  
 statistical summary products are produced in a common grid based upon 
 Australian Albers (Albers Collection 3 grid). This page
 describes how we developed this grid and how it differs from the
 previous Australian Albers grid (Albers Collection 2 grid).
 
-## How did we develop the new Collection 3 grid?
+## How did we develop the new Collection 3 grids?
 
-In developing this new grid, we had several goals:
+Landsat and Sentinel derivative products are built using two distinct grids.
 
-* be compatible with the new resolution of Landsat ($30m^2$ from
-   $25m^2$)
-* be interoperable between both Landsat ($30m^2$) and Sentinel-2
-   ($10m^2$, $20m^2$ and $60m^2$)
-* have a naming convention that was non-negative, removing the need for
-   plus and minus symbols that can be problematic for some software applications
+We call these:
+* DEA Landsat Summary Product Grid (Collection 3) and,
+* DEA Sentinel Summary Product Grid (Collection 3)
 
-The tile size is defined as $96km^2$. For Landsat, this will result
-in a raster size of $3200 \times 3200 pixels$ of $30m^2$. For 
-Sentinel-2, this will result in a raster size of $9600 \times 9600 pixels$ 
-of $10m^2$.
+The Landsat tile size is defined as $96km^2$. This results
+in a raster size of $3200 \times 3200 pixels$ of $30m^2$. 
+
+For Sentinel based products, the tile size is $32km^2$ this results in a 
```

Comment:
```suggestion
For Sentinel-2 products, the tile size is $32km^2$ which results in a 
```

Reply:
So I explicilty left it as `Sentinel` instead of `Sentinel-2` in case we decide to use this grid for Sentinel-1 based derivatives.